package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "real-world-vue",
```

Comment:
Can we update the name here?

Reply:
Sorry, it was from a tutorial I was following